### PR TITLE
TCES-10: Implement backend testing libraries

### PR DESCRIPTION
**Pull request by thardchi**
# Description

Link to ticket: https://github.com/uoftblueprint/tces/issues/10

Installed jest library and wrote a test to verify that the response from the app.get endpoint is "Hello from server!" and the expected status code is 200.

New devDependencies:
        "@types/jest": "^29.2.3",
        "jest": "^29.3.1"

Fixes # 10

## Sample curl
Ignore if PR does not affect an endpoint.
## Response
Ignore if PR does not affect an endpoint.

## Type of change

# Checklist:

- [ ] I have run `npm run lint:fix`
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (Delete if frontend)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,11 @@
         "build": "cd client && npm run build",
         "install:all": "npm i && cd client && npm i && cd ../server && npm i",
         "install:client": "cd client && npm i",
-        "install:server": "cd server && npm i"
+        "install:server": "cd server && npm i",
+        "test": "jest"
```
Comment (EclipseIsDead) [inline package.json:9]: should probably only have jest in server level package.json

**Review by EclipseIsDead — CHANGES_REQUESTED** [review]

**Review by annie951977 — CHANGES_REQUESTED** [review]
I agree with Siddarth, looks good otherwise

**Review by EclipseIsDead — APPROVED** [review]
lgtm

**Review by annie951977 — APPROVED** [review]
pog

**Pull request merged**